### PR TITLE
Fixed #253 bug

### LINKED ISSUE #253: Collapse widget issue
### What steps will reproduce the problem?
echo Collapse widget

### What is the expected result?
Collapsed headings should contain `collapsed` class in `a` tag

### What do you get instead?
The class appears in the tag only after any of the panels is opened

### Additional info
I replaced this:
```php
$itemToggleOptions = array_merge([
    'tag' => 'a',
    'data-toggle' => 'collapse',
], $this->itemToggleOptions);
Html::addCssClass($itemToggleOptions, ['widget' => 'collapse-toggle']);
```

with the following code
```php
$active = false;
if (isset($options['class'])) {
    $classes = is_string($options['class']) ? explode(' ', $options['class']) : $options['class'];
    $active = in_array('in', $classes);
}

$itemToggleOptions = array_merge([
    'tag' => 'a',
    'data-toggle' => 'collapse',
], $this->itemToggleOptions);
Html::addCssClass($itemToggleOptions, ['widget' => 'collapse-toggle']);

if (!$active) {
    Html::addCssClass($itemToggleOptions, ['collapsed' => 'collapsed']);
}
```
and now it works like a charm

| Q                | A
| ---------------- | ---
| Yii version      | 2.0.17
| PHP version      | 7.3.2
| Operating system | Mac OS X Mojave

### PR DESCRIPTION
**Pull request by evil1**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #253

### CODE REVIEW

**Comment by evil1** [comment]
@samdark Any enhancements required from my side?

File: src/Collapse.php [hunk 1]
```diff
@@ -184,12 +184,22 @@ public function renderItem($header, $item, $index)
                 $header = Html::encode($header);
             }
 
+            $active = false;
+            if (isset($options['class'])) {
+                $classes = is_string($options['class']) ? explode(' ', $options['class']) : $options['class'];
```
Comment (samdark) [inline src/Collapse.php]: Doing it like that will behave well if one will use extra spaces while setting options:

```suggestion
                $classes = is_string($options['class']) ? preg_split('/\s+/', $options['class'], -1, PREG_SPLIT_NO_EMPTY) : $options['class'];
```

File: src/Collapse.php [hunk 2]
```diff
@@ -184,12 +184,22 @@ public function renderItem($header, $item, $index)
...
+                $active = in_array('in', $classes);
```
Comment (samdark) [inline src/Collapse.php]:
```suggestion
                $active = in_array('in', $classes, true);
```

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**